number_theory/prime.h

```diff
@@ -149,6 +150,41 @@ class EulerSieve {
   }
 };
 
+// Generates all coprime pairs of integers under |num_limit| (inclusive).
+// Each pair (x, y) in the results should satisfy num_limit >= x >= y >= 0.
```

Comment:
Why do we list pairs (x, y) such that x >= y? Isn't x <= y more intuitive?

Reply:
I am not sure if x>=y or x<=y is more intuitive. Because the algorithm on wiki (see comments below) uses x>=y.